components/eventing-controller/reconciler/subscription/reconciler_test.go

```diff
@@ -122,6 +121,11 @@ var _ = Describe("Subscription Reconciliation Tests", func() {
 				Type:    v1.EventTypeWarning,
 			}
 			getK8sEvents(&subscriptionEvents, givenSubscription.Namespace).Should(reconcilertesting.HaveEvent(subscriptionDeletedEvent))
+
+			By("Sending valid webhook requests only")
+			validRequests, invalidRequests := countRequestsForWebhooks(reconcilertesting.EventType, reconcilertesting.EventTypeNotClean)
+			Expect(validRequests == 0).Should(BeTrue())
+			Expect(invalidRequests == 0).Should(BeTrue())
```

Comment:
You may consider, `Expect(validRequests).Should(Equal(0))` which is more Gomega-ish IMO

Reply:
Done